docs/source/testing.rst

```diff
@@ -97,41 +97,112 @@ Smoke test:
+    File "/Users/nbonilla/Documents/Koku/koku/koku/api/provider/test/tests_models.py", line 43, in test_delete_single_provider_skips_delete_archived_data_if_customer_is_none
     self.aws_provider.delete()
-AssertionError: no logs of level WARNING or higher triggered on api.provider.models
-
-----------------------------------------------------------------------
-Ran 756 tests in 2025.258s
-FAILED (failures=1)
-```
-
-# Gotchas:
-- The logger is disabled by default during unit tests. If you are building a unit test that asserts that a log occurs, you must re-enable the logger. For example:
-
-```
-import logging
-with self.assertLogs(logger='masu.external.downloader.aws.aws_report_downloader', level='WARN') as cm:
-            logging.disable(logging.NOTSET)
-            self.aws_report_downloader._remove_manifest_file("None")
-            self.assertEqual(['WARN: Could not delete manifest file at'], cm.output)
-```
+    AssertionError: no logs of level WARNING or higher triggered on api.provider.models
+    
+
+==========
+PDB in IQE
+==========
+Begin a shell session into the docker container that runs IQE by running the following command::
+
+    koku/testing/run_test.sh bash
+
+since IQE itself is installed in this container, you may run the following command while inside the container to run a specific test::
+
+    iqe tests plugin hccm -k test_api_aws_storage_filtered_top --pdb
+
+or to run all tests::
+
+    iqe tests plugin hccm --pdb
+
+Any test that fails should start a PDB session.
+
+=========================
+Unit testing log messages
+=========================
+The logger is disabled by default during unit tests. If you are building a unit test that asserts log messages, you must re-enable the logger. For example::
+
+    import logging
+    with self.assertLogs(logger='masu.external.downloader.aws.aws_report_downloader', 
+    level='WARN') as cm:
+                logging.disable(logging.NOTSET)
+                self.aws_report_downloader._remove_manifest_file("None")
+                self.assertEqual(['WARN: Could not delete manifest file at'], cm.output)
+                
+                
+===
+IQE
+===
+
+Prerequisites:
+    - koku is deployed via docker-compose or other method, 
+    - you are connected to the Red Hat internal network.
+
+For simple setup, continue to the prerequisites below, otherwise, for information on IQE, visit: https://gitlab.cee.redhat.com/insights-qe/hccm-plugin/tree/master 
+
+===================
+Running IQE locally
+===================
+
+IQE expects the database to be empty.
+To run IQE Smoke, Vortex or API tests, run one of the following commands, respectively::
+
+    make docker-iqe-smokes-tests
+    make docker-iqe-vortex-tests
+    make docker-iqe-api-tests
+
+
+====================
+Running PDB in IQE
+====================
```

Comment:
So, the extra = here are causing the the title bold syntax to bleed into the line below. Same comment as above.